src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -45,16 +70,7 @@ function PageCard({
         <Card.Title className="d-flex mb-0 align-items-center justify-content-between">
           <h4 className="m-0 p-0">{page.name}</h4>
           {(page.allowedOperations.configure || page.allowedOperations.enable)
-          && (
-            <IconButton
-              className="mb-0 mr-1"
-              src={Settings}
-              iconAs={Icon}
-              size="inline"
-              alt={intl.formatMessage(messages.settings)}
-              onClick={() => history.push(`${pagesAndResourcesPath}/${page.id}/settings`)}
-            />
-          )}
+          && settingsButton()}
```

Comment:
I disagree that we need to bound "external url direction" with `allowedOperations`. Currently the behavior is bound to show `->` Iff certain param in allowed operations are enabled. Which should not be the case for the app who uses external links. 

I would recommend to decouple that logic.